netket/optimizer/stochastic_reconfiguration.py

```diff
@@ -0,0 +1,303 @@
+import numpy as _np
+from scipy.linalg import lstsq as _lstsq
+from scipy.sparse.linalg import LinearOperator
+from netket.stats import sum_on_nodes as _sum_on_nodes
+from scipy.sparse.linalg import cg, gmres, minres
+from mpi4py import MPI
+
+
+class SR:
+    r"""
+    Performs stochastic reconfiguration (SR) updates.
+    """
+
+    def __init__(
+        self,
+        lsq_solver=None,
+        diag_shift=0.01,
```

Comment:
Since we seem to be changing defaults already: Isn't a shift of 0.01 a bit large as default? Maybe something like `1e-4` would be more appropriate?

Reply:
For most of the examples we have a shift of 1e-4 would be too small...